packages/protocol/src/interaction/FabricAccessControlManager.ts

```diff
@@ -59,10 +62,11 @@ export class AccessDeniedError extends StatusResponseError {
 }
 
 /**
- * Implements Access Control Logic as per Matter Specification @see {@link MatterSpecification.v12.Core} § 6.6.5.2.
+ * Implements Access Control Logic For one fabric as per Matter Specification @see {@link MatterSpecification.v12.Core} § 6.6.5.2.
  */
-export class AccessControlManager {
-    #aclList: AclList;
+export class FabricAccessControlManager {
```

Comment:
Since we're renaming anyway, WDYT about taking "Manager" off the name?  Would make it shorter and I'm not sure it adds much

Reply:
ok